homeassistant/components/fritzbox_callmonitor/sensor.py

```diff
@@ -56,18 +56,15 @@ async def async_setup_entry(
         FRITZBOX_PHONEBOOK
     ]
 
-    phonebook_name: str = config_entry.title
     phonebook_id: int = config_entry.data[CONF_PHONEBOOK]
     prefixes: list[str] | None = config_entry.options.get(CONF_PREFIXES)
     serial_number: str = config_entry.data[SERIAL_NUMBER]
     host: str = config_entry.data[CONF_HOST]
     port: int = config_entry.data[CONF_PORT]
 
-    name = f"{fritzbox_phonebook.fph.modelname} Call Monitor {phonebook_name}"
```

Comment:
This change would mean that none of the entities will have a phonebook name. Can one phone (which is what fritzbox_phonebook.fph.modelname is, I think) have more phonebooks?

Reply:
It can, you have to add another device and chose another phonebook there:
<img width="694" alt="image" src="https://github.com/home-assistant/core/assets/271149/3e16b531-5df4-47db-84bc-f998ed1bd9c0">

I did not find any information how to append the phonebook name to the entity name, though.

Reply:
Currently we can't and there are some discussion on adding more variables to entity names. But I think with this change, you would get `sensor.something_call_monitor` and `sensor.something_call_monitor_2` unless you rename the device.

Maybe @cdce8p has an idea for now without placeholders that would make this correct. Otherwise you might need to wait with this PR :(

Reply:
Yes, second one is `fritz_box_7590_call_monitor_2`.

Reply:
Changed to use new translation placeholders.